Lib/urllib/request.py

```diff
@@ -799,6 +799,7 @@ def __init__(self, proxies=None):
         assert hasattr(proxies, 'keys'), "proxies must be a mapping"
         self.proxies = proxies
         for type, url in proxies.items():
+            type = type.lower()
```

Comment:
Is it worth adding this behavior change in https://docs.python.org/3/library/urllib.request.html#urllib.request.ProxyHandler with a `versionchanged` directive note?

Reply:
For what it's worth, I don't think that this change needs to be documented, but it would be nice to see what a core dev thinks.